docs/build/smart-contracts/verify/hardhat.md

```diff
@@ -49,7 +49,7 @@ module.exports = {
   },
   etherscan: {
     apiKey: {
-      klaytn: "unnecessary",
+      kairos: "unnecessary",
```

Comment:
@youngancient Thanks for your contribution.
Did you test it with explorer?

Reply:
Yes, I did. My first step was to deploy a contract on kaia testnet (kairos) and then verify.
My hardhat config at first looked like this:
```
import { HardhatUserConfig } from "hardhat/config";
import "@nomicfoundation/hardhat-toolbox";

require('dotenv').config();

const config: HardhatUserConfig = {
  solidity: "0.8.20",
  networks: {
    // for testnet
    kairos: {
      url: process.env.VITE_KAIA_TESTNET_RPC_URL!,
      accounts: [process.env.PRIVATE_KEY!],
      // gasPrice: 100000000000,
    },
  },
  etherscan: {
    apiKey: {
      klaytn: "unnecessary",
    },
    customChains: [
      {
        network: "kairos",
        chainId: 1001,
        urls: {
          apiURL: "https://api-baobab.klaytnscope.com/api",
          browserURL: "https://kairos.kaiascope.com",
        },
      },
    ],
  },
};

export default config;

``` 
In the network object, the network name is "kairos" but in the etherscan apiKey obj, "klaytn" was there, so I had the bug which was fixed by changing klaytn to kairos.

[The verified contract](https://kairos.kaiascope.com/account/0x33651f2fc249f2dA9903b46f934e76cD79181ED8?tabId=eventLog)